exercism/bowling/bowling.js

```diff
@@ -0,0 +1,62 @@
+var Bowling = function(rolls){
+  this.rolls = rolls
+}
+
+Bowling.prototype.score = function() {
+  var total = 0
+  var totalFrameScore = 0
+  var numberOfRollsInFrame = 0
+
+  for(var i = 0; i < this.rolls.length; i++) {
+     total += this.rolls[i]
+     numberOfRollsInFrame++
+     totalFrameScore += this.rolls[i]
+
+     if (numberOfRollsInFrame === 2 && totalFrameScore === 10) {
```

Comment:
One suggestion, you could pull out " numberOfRollsInFrame === 2 && totalFrameScore === 10 "
and put it in a function named something like isSpare.

Bowling.isSpare = function( numberOfRollsInFrame, totalFreamScore )  {
    return numberOfRollsInFrame === 2 && totalFrameScore === 10
}

 This would eliminate the need for comments telling which is which.

if ( Bowling.isSpare( numberOfRollsInFrame, totalFrameScore ) ) {
    // Do stuff
}